specification/communication/data-plane/PhoneNumbers/preview/2021-10-25-preview/phonenumbers.json

```diff
@@ -0,0 +1,1639 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PhoneNumbersClient",
+    "description": "The phone numbers client uses Azure Communication Services to purchase and manage phone numbers and short codes.",
+    "version": "2021-10-25-preview"
+  },
+  "paths": {
+    "/availablePhoneNumbers/countries/{countryCode}/:search": {
+      "post": {
+        "tags": [
+          "Find and purchase available phone numbers"
+        ],
+        "summary": "Search for available phone numbers to purchase.",
+        "operationId": "PhoneNumbers_SearchAvailablePhoneNumbers",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "countryCode",
+            "description": "The ISO 3166-2 country code, e.g. US.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The phone number search request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PhoneNumberSearchRequest"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The service has accepted the request and will start processing later. It will return 'Accepted' immediately and include an 'Operation-Location' and 'Location' headers. Client side should further query the operation status using the URL specified in 'Operation-Location' header. When operation completes successfully query final result using URL specified in 'Location' header.",
+            "schema": {
+              "$ref": "#/definitions/PhoneNumberSearchResult"
+            },
+            "headers": {
+              "Location": {
+                "description": "URL to retrieve the final result after operation completes.",
+                "type": "string"
+              },
+              "Operation-Location": {
+                "description": "URL to query for status of the operation.",
+                "type": "string"
+              },
+              "operation-id": {
+                "description": "The operation id.",
+                "type": "string"
+              },
+              "search-id": {
+                "description": "The search operation id.",
```

Comment:
How does this differ from the `operation-id`?  Can we add enough to the descriptions so customers can determine when to use one or the other?

Reply:
This section of the swagger is part existing endpoints that have already been GAd and it is not changing. We are bringing these endpoints into the new version that is entering Public Preview and adding ShortCodes functionality.

Regarding your specific question about the difference, search_id is an identifier for the search, I think it is a Guid, while operation_id is a string with the format 'search_{search_id}'. This allows different long running operations to track their status thru the same /operations/{operation_id} endpoint with no operation_id collisions between different operations. I think that was the intent, however, like I explained before, this is existing functionality that has been GAd and is not changing on this new version.